keylime/cloud_verifier_tornado.py

```diff
@@ -421,6 +422,21 @@ def post(self):
                     logger.warning("POST returning 400 response. Expected non zero content length.")
                 else:
                     json_body = json.loads(self.request.body)
+
+                    # Handle allowlists
+                    allowlist_bundle = json.loads(json_body.get("ima_policy"))
+                    if allowlist_bundle:
+                        try:
+                            allowlist = ima.unbundle_allowlist(allowlist_bundle)
```

Comment:
@mpeters I think I agree with `allowlist_bundle` being renamed to `ima_policy_bundle`, but the `allowlist` variable here really is just the allowlist, not a full IMA policy. On the next line, it's combined with the exclusion list found in `allowlist_bundle` (soon to be `ima_policy_bundle`) to create the final `ima_policy`.

Reply:
@mpeters I've pushed an update with naming changes - giving everything another read, I think the `process_allowlists` function is the most confusing, since that takes in both an allowlist and exclusion list and spits out a combined "IMA policy". I've renamed it `process_ima_policy` accordingly.

Let me know how it reads now.